android-core/src/main/kotlin/com.mparticle/NetworkUtilities.kt

```diff
@@ -0,0 +1,26 @@
+package com.mparticle
+
+object NetworkUtilities {
```

Comment:
let's get rid of this class and include the `getUrlWithPrefix` in the api client implementation